fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/system/ChunkReader.java

```diff
@@ -210,10 +210,13 @@ public Object readItem() throws Exception {
         searchContext.setPageNumber(pageNum);
         List<Resource> resources = null;
         FHIRTransactionHelper txn = new FHIRTransactionHelper(fhirPersistence.getTransaction());
-        txn.enroll();
-        persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
-        resources = fhirPersistence.search(persistenceContext, resourceType).getResource();
-        txn.unenroll();
+        txn.begin();
+        try {
+            persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
+            resources = fhirPersistence.search(persistenceContext, resourceType).getResource();
+        } finally {
+            txn.end();
+        }
```

Comment:
not sure if this can work well, need to be verified by the minio enabled build pipeline.